packages/adminrouter/extra/src/lib/auth/open.lua

```diff
@@ -108,9 +108,13 @@ function _M.init(use_auth)
         return res.do_authn_and_authz_or_exit()
     end
 
-    -- /metadata
     -- /pkgpanda/active.buildinfo.full.json
     -- /dcos-metadata/
+    res.access_misc_metadata_endpoint = function()
+        return res.do_authn_and_authz_or_exit()
```

Comment:
This function call hints it does the `authz`.

Reply:
In open authn is also authz.

https://github.com/dcos/dcos/blob/3847b7e21047994c84cac58aabed09d39b9206fd/packages/adminrouter/extra/src/lib/auth/open.lua#L42-L43